app/Http/Controllers/Tabbar/Navigation/Contents/ContentsController.php

```diff
@@ -3,30 +3,21 @@
 namespace App\Http\Controllers\Tabbar\Navigation\Contents;
 
 use App\Http\Controllers\Controller;
-use App\Settings\PathInfo;
-use Parsedown;
 
-class ContentsController extends Controller
-{
-    /*
-    |--------------------------------------------------------------------------
-    | ContentsController
-    |--------------------------------------------------------------------------
-    |
-    | Contentsコントローラ
-    |
-    */
+/**
+ * ContentsController
+ * 
+ * contentsタブ内で表示するコンテンツを扱う
+ */
+class ContentsController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition